examples/ablation/log.12Nov24.mpi_1.ablation.multi.inner.3d

```diff
@@ -0,0 +1,267 @@
+variable yh equal 10.0
+variable zh equal 0.5
+
+variable Nx equal 44
+variable Ny equal 20
+variable Nz equal 1
+
+boundary		p p p
+
+create_box		${xl} ${xh} ${yl} ${yh} ${zl} ${zh}
+create_box		0 ${xh} ${yl} ${yh} ${zl} ${zh}
+create_box		0 22 ${yl} ${yh} ${zl} ${zh}
+create_box		0 22 0 ${yh} ${zl} ${zh}
+create_box		0 22 0 10 ${zl} ${zh}
+create_box		0 22 0 10 -0.5 ${zh}
+create_box		0 22 0 10 -0.5 0.5
+Created orthogonal box = (0 0 -0.5) to (22 10 0.5)
+create_grid		${Nx} ${Ny} ${Nz}
+create_grid		44 ${Ny} ${Nz}
+create_grid		44 20 ${Nz}
+create_grid		44 20 1
+Created 880 child grid cells
+  CPU time = 0.000800473 secs
+  create/ghost percent = 82.2424 17.7576
+balance_grid  rcb cell
+Balance grid migrated 0 cells
+  CPU time = 0.000161832 secs
+  reassign/sort/migrate/ghost percent = 62.3906 0.860769 10.9947 25.7539
+
+global      nrho 1.0 fnum 0.001
+global      surfs explicit/distributed
+
+species			air.species O2
+mixture			air O2 vstream 0.0 0 0
+
+read_surf               data.etch2d
+  8 lines
+  0.25 21.25 xlo xhi
+  0.25 8.25 ylo yhi
+  0 0 zlo zhi
+  0.5 min line length
+  118 0 = cells overlapping surfs, overlap cells with unmarked corner pts
+  149 613 118 = cells outside/inside/overlapping surfs
+  118 = surf cells with 1,2,etc splits
+  52.25 52.25 = cell-wise and global flow volume
+  CPU time = 0.000533935 secs
+  read/check/sort/surf2grid/ghost/inout/particle percent = 7.1151 8.80969 3.0661 68.1034 12.9057 8.04405 0.0189162
+  surf2grid time = 0.000363628 secs
+  map/comm1/comm2/comm3/comm4/split percent = 37.793 5.05297 21.6229 3.19887 19.3497 9.37579
+surf_collide            1 diffuse 300.0 0.0
+surf_modify             all collide 1
+
+fix			                fablate ablate all 1 1 uniform 1 multiple yes
+create_isurf            all fablate 100.5 multi
+  880 0 0 = cells outside/inside/overlapping surfs
+  0 = surf cells with 1,2,etc splits
+  220 220 = cell-wise and global flow volume
+  0.25 21.25 xlo xhi
+  0.25 8.25 ylo yhi
+  0 0 zlo zhi
+  0.353553 min line length
+  118 = cells with surfs
+  118 = total surfs in all grid cells
+  1 = max surfs in one grid cell
+  0.707107 = min surf-size/cell-size ratio
+  118 0 = cells overlapping surfs, overlap cells with unmarked corner pts
+  149 613 118 = cells outside/inside/overlapping surfs
+  118 = surf cells with 1,2,etc splits
+  52.375 52.375 = cell-wise and global flow volume
+
+timestep		0.001
+
+#compute 1 property/grid all vol
+#compute 1r reduce sum c_1[*]
+
+#variable vout equal c_1r
+#fix massr print 5 "${vout}" append vol_inner_multi.dat screen no
+
+stats			1
+stats_style		step cpu np nattempt ncoll nscoll nscheck
+
+run 100
+Memory usage per proc in Mbytes:
+  particles (ave,min,max) = 0 0 0
+  grid      (ave,min,max) = 1.51379 1.51379 1.51379
+  surf      (ave,min,max) = 0.0117035 0.0117035 0.0117035
+  total     (ave,min,max) = 1.86455 1.86455 1.86455
+Step CPU Np Natt Ncoll Nscoll Nscheck 
+       0            0        0        0        0        0        0 
```

Comment:
@andrewykhong this example has all zeros for stats output. Is there some variable we could use to check the ablation is working? Can use `compute reduce ave` to output a single value to the stats.

Reply:
I'll change the stats style to show the total volume (there's a compute reduce with the volumes) and comment once more after I'm done

Reply:
@andrewykhong sounds good, thanks

Reply:
updated the stats_style so it outputs the total gas volume over time @stanmoore1

Reply:
Thanks!